repos/system_upgrade/el7toel8/actors/storagescanner/actor.py

```diff
@@ -30,7 +30,9 @@ def get_cmd_output(self, cmd, delim, expected_len):
             raise StopIteration()
 
         try:
-            output = subprocess.check_output(cmd)
+            # when there is any fd except 0,1,2 open, lvm closes the fd and prints a warning.
+            # In our case /dev/urandom has other fd opened, probably for caching purposes.
+            output = subprocess.check_output(cmd, env={'LVM_SUPPRESS_FD_WARNINGS': '1', 'PATH': os.environ['PATH']})
```

Comment:
Yes, that's good solution. It's kinda regular message. The same thing we did in legacy project so agree with the solution. LGTM. I am going to test this now.

Reply:
Tested.  works as expected.